source/renderer/app/api/utils/requestBinary.js

```diff
@@ -0,0 +1,100 @@
+// @flow
```

Comment:
Do we really need this `requestBinary`? We should rather use the existing `request` with modifications if they are needed.